### ISSUE TITLE
vim.opt.winbar format is ignored if symbol_in_winbar is enabled

### ISSUE THREAD

**Issue by halfdan** [glepnir/lspsaga.nvim#324]
### Description
vim.opt.winbar format is ignored if symbol_in_winbar is enabled

**Expected Behavior**
I expect LspSaga to just add the symbol to the winbar while keeping my original winbar format (`%=%m %f`) intact.

**Actual Behavior**
LspSaga overrides the original format and instead displays the symbol + `vim.fn.expand('%:t')`

### Details

<details><summary>Reproduce</summary>

1. Set `vim.opt.winbar = "%=%m %f"`
2. Set `symbol_in_winbar` to true
</details>

<details><summary>Environment</summary>

- nvim --version output: latest nightly
- Operating system: Mac 
- lspsaga commit: master/latest

</details>

**Comment by glepnir** [comment]
@halfdan yup this is problem.  I just find a way that add the symbols into tail..

```lua
local opt_winbar = vim.opt.winbar:get()
winbar_val = opt_winbar .. winbar_val
api.nvim_win_set_option(current_win,'winbar',winbar_val)
```

**Comment by glepnir** [comment]
I think I know how to do it.

**Comment by glepnir** [comment]
support
